docs: update FTS docs for JS SDK (#1634)
lancedb/lancedb: docs/src/fts.md

LanceDB provides support for full-text search via Lance (before via [Tantivy](https://github.com/quickwit-oss/tantivy) (Python only)), allowing you to incorporate keyword-based search (based on BM25) in your retrieval solutions.

> Currently, the Lance full text search is missing some features that are in the Tantivy full text search. This includes query parser and customizing the tokenizer. Thus, in Python, Tantivy is still the default way to do full text search and many of the instructions below apply just to Tantivy-based indices.

Reviewer: ```suggestion
Currently, the Lance full text search is missing some features that are in the Tantivy full text search such as customizing the tokenizer and advanced query parsing. Thus, in Python, Tantivy is still the default way to do full text search and many of the instructions below apply just to Tantivy-based indices.
```
